client/lib/trigram/test/index.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * Internal dependencies
+ */
+import { cosineSimilarity, trigrams, gramsToLookup, lookupToMagnitude } from '../';
+
+describe( 'trigram: cosineSimilarity()', () => {
+	test( 'Equal Strings should have cosineSimilarity of ~1', () => {
+		const str1 = 'This is a string.';
+		const str2 = 'This is a string.';
+		const sim = cosineSimilarity( str1, str2 );
+		expect( sim ).toBeGreaterThan( 0.99 );
+		expect( sim ).toBeLessThan( 1.01 );
```

Comment:
@mreishus Jest [has support for approximate float comparisons](https://jestjs.io/docs/expect#tobeclosetonumber-numdigits) like this:
```js
expect( sim ).toBeCloseTo( 1.0 );
```
The default "closeness" threshold is 0.01, so you don't even need to specify a custom one.